uniffi_aries_vcx/README.md

```diff
@@ -1,24 +1,55 @@
 # UniFFI Aries VCX Proof Of Concept
+
 This sub-directory contains the UniFFI wrappers and tools to generate Kotlin & Swift mobile wrappers over the `aries-vcx` crate.
 
 ## Core
+
 Core contains a crate which wraps over the `aries-vcx` crate to provide a UniFFI-ready interface for wrapping. This crate's interface is what is used in `demos`.
 
 ## Demos
+
 Coming soon..
 
 ## Building
+
 This module is distributed in three directories.
+
 1. `core`: The UniFFI bindings for Aries VCX.
 2. `demo`: Demo application consuming the UniFFI wrapper code.
 3. `scripts`: Contains helper sh scripts to ease the process of setting up, building and generating the UniFFI code.
 
-To set up the Android SDK, NDK and bootstrap the demo application, you can simply run the following scripts in the order:
-1. `android.prepare.sh`: This script will prepare the Android SDK and NDK for your system.
-2. `android.toolchain.sh`: This script will prepare the required android toolchain.
-3. `android.build.sh`: This script will build the UniFFI bindings and bootstrap the demo application for the target architecture.
+To generate the UniFFI bindings, you need to set up [cargo-ndk](https://github.com/bbqsrc/cargo-ndk) on your system. Instructions to set up `cargo-ndk` are available [here](https://github.com/bbqsrc/cargo-ndk#installing), but can be summed up as:
+
+1. Install `cargo-ndk`.
+
+```bash
+cargo install cargo-ndk
+```
+
+2. Add target architectures you want to use.
+
+```bash
+rustup target add <target>
+```
+
+eg. for Android ARM64 on host linux platform.
+
+```bash
+rustup target add aarch64-linux-android
+```
+
+More documentation can be found [here](https://rust-lang.github.io/rustup/cross-compilation.html). Information on supported platforms can be found [here](https://doc.rust-lang.org/nightly/rustc/platform-support.html).
+
+3. Make sure your ANDROID_SDK and ANDROID_NDK_HOME environment variables point to the SDK and NDK roots respectively.
```

Comment:
Maybe just change this to: 
> if cargo-ndk cannot find the NDK installed on your machine, you may have to configure environment variables as described here: https://github.com/bbqsrc/cargo-ndk#usage

Reply:
since most users won't have to configure environment variables if they installed with android studio (as we recommend)